Tools/PrivateRepoLib.ps1

```diff
@@ -0,0 +1,113 @@
+
+
+Function New-WinGetManifest
+{
+    <#
+    .SYNOPSIS
+    Submits Manifest files to the Azure Private REpository
+    
+    .DESCRIPTION
+    By running this function with the required inputs, it will connect to the Azure Tennant that hosts the Windows Package Manager Private Repository, then collects the required URL for Manifest submission before retrieving the contents of the Manifest JSON to submit.
+    
+    .PARAMETER PrivateRepoName
+    Name of the Windows Package Manager Private repository. Can be identified by running: "winget source list" and using the Repository Name
```

Comment:
not sure why you need this to generate a push to the packagemanifest endpoint? looking..

Reply:
I've split this function to have two sets of parameters to be accepted. The executing user only requires either the Windows Package Manager private source name, or the Azure information to determine the URL for publishing to.

Happy to discuss this in more detail.